artman/cli/main.py

```diff
@@ -113,6 +111,14 @@ def parse_args(*args):
              '/path/to/config.yaml:config_section_A|config_section_B. '
              'This is an advanced feature; for most cases, use `--api`.',
     )
+    api.add_argument('--batch',
+        action='store_true',
+        help='Enables batch mode, which will generate all APIs for the '
```

Comment:
We did have this. It worked in a similar way to how it works with APIs now - the batch pipeline took tasks from each language pipeline.

I have not tried to support it now - I do not think it adds as much value as the API-based batch. Even if we had it, my feeling is that language owners would still want to be responsible for refreshing their own languages, at least in the short term. In the long term, this could all (or nearly all) be automated away by tools that don't rely on a language batch mode.